cpp/include/Ice/Config.h

```diff
@@ -28,7 +28,7 @@
 #endif
 
 #ifndef ICE_API
-#    if defined(ICE_STATIC_LIBS)
+#    if defined(ICE_BUILDING_SLICE_COMPILERS) // Only defined for Windows builds.
```

Comment:
On Linux and macOS, ICE_DECLSPEC_EXPORT == ICE_DECLSPEC_IMPORT and is actually the default, which is fine for static builds too.